src/tests/ftest/cart/protocol_info.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * (C) Copyright 2018-2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+
```

Comment:
would be good to have a comment here describing the purpose or what this test does